pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>OfflineReverseGeocode</groupId>
```

Comment:
I think if we're pulling this in and maintaining proprietary pieces on top of it, it should become part of our own namespace

Reply:
Yeah I was wondering how we should handle that. I'll check and see how that hammock one is done.

Reply:
I checked a bunch of other repositories that we've forked to sportinginnovations, but they all left the groupId the same as it was before.

Reply:
that may be true, but I'm not sure we care to expose or rely upon the original source naming like that. In fact I think it would be strictly incorrect to use the _exact same_ package name and group information, as it is not the same and theoretically could have versioning conflicts with the real library.  At the very least we should add -FTS or some equivalent... I'd think we should just translate the whole thing over

Reply:
Looked around at standards for forking repositories. It was suggested that changing the groupId, name, and description are a good idea, so I've updated those three items. I picked `com.fanthreesixty.libraries` as the groupId - let me know if something else would be better.